### PR TITLE
Add Category pages

### PR DESCRIPTION
**Pull request by rahulcs754**
Added Category Page
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,15 +27,15 @@ <h1 class="header-logo"><a href="#">Coding Quiz</a></h1>
               <li><a href="#"> Home </a></li>
               <li><a href="#"> About </a></li>
               <li><a href="#"> Faq </a></li>
-             
             </ul>
         </header>
```
Comment (shubhamkumar648) [inline index.html:32]: avoid space

File: index.html [hunk 2]
```diff
@@ -34,6 +34,48 @@ <h1 class="header-logo"><a href="#">Coding Quiz</a></h1>
...
+                <option value="">Easy</option>
+            </select>
+           
+        </div>
+    </div>
+</div>
+
+
+
     <div class="section flex flex-row">
```
Comment (shubhamkumar648) [inline index.html]: same here

File: assets/css/custom.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /* color palattes used in this project */
 :root{
     --white-color: #EEEEEE;
```
Comment (shubhamkumar648) [inline assets/css/custom.css:3]: tou can also use in hsl

**Review by shubhamkumar648 — APPROVED** [review]
just minnor change , reamaing look good to me !

**Review by tanishq20 — APPROVED** [review]
Good class names. Easy to understand.

**Pull request merged**